ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-addon-interop__v4.13-IIB.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  rhods-addon-tests:
+    name: ods-ci
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: smoke-rosa-rhods-addon-baseline-fips
+  cron: 0 0 1 1 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      ADDON1_CONFIG: name=managed-odh;notification-email=interop-qe-ms@redhat.com
+      CLUSTER_NAME: ods-ad-all-413
```

Comment:
@myakove @rnetser maybe we should have different cluster name for IIB/GA cases?
This way both jobs could be triggered together

Reply:
@omrirh please remove `CLUSTER_NAME` from our tests